internal/services/network/application_gateway_resource.go

```diff
@@ -409,6 +409,24 @@ func resourceApplicationGateway() *pluginsdk.Resource {
 				},
 			},
 
+			"global_configuration": {
```

Comment:
configuration is redundant
```suggestion
			"global": {
```

Reply:
i recommend reverting this change, although it is not a blocker for me.

the name `global_configuration` matches the nomenclature used by Azure resource manager (`globalConfiguration`). adopting their nomenclature, even if somewhat redundant, seems useful for users of this resource. it also matches the convention established by other `*_configuration`s, such as `frontend_ip_configuration` (`frontendIPConfiguration` in ARM)

using the name `global_configuration` keeps the naming consistent with ARM and consistent within this resource